search-service/src/test/kotlin/com/egm/stellio/search/entity/service/EntityServiceQueriesTests.kt

```diff
@@ -281,8 +297,8 @@ class EntityServiceQueriesTests : WithTimescaleContainer, WithKafkaContainer {
         "listOfString==\"iot\", 1, urn:ngsi-ld:BeeHive:01",
         "listOfString==\"data's processing\", 1, urn:ngsi-ld:BeeHive:01",
         "listOfString==\"stellio\", 2, 'urn:ngsi-ld:BeeHive:01,urn:ngsi-ld:BeeHive:02'",
```

Comment:
why not use multiline string here too since `"` is used and needs to be escaped too?